setup.py

```diff
@@ -28,7 +28,7 @@
     entry_points={
         # define console_scripts here, see setuptools docs for details.
         'console_scripts': [
-            'pdfmorph = diffpy.pdfmorph.pdfmorphapp:main',
+            'stretched_nmf = diffpy.snmf.stretchednmfapp:main',
```

Comment:
maybe we could call it `snmf` which is less typing and maybe unique enough?

Reply:
Ok. I will change it